Sources/ArcGISToolkit/Components/BasemapGallery/BasemapGallery.swift

```diff
@@ -54,19 +54,19 @@ public struct BasemapGallery: View {
         portal: Portal,
         geoModel: GeoModel? = nil
     ) {
-        viewModel = BasemapGalleryViewModel(geoModel, portal: portal)
+        _viewModel = StateObject(wrappedValue: BasemapGalleryViewModel(geoModel, portal: portal))
     }
     
     /// The view model used by the view. The `BasemapGalleryViewModel` manages the state
     /// of the `BasemapGallery`. The view observes `BasemapGalleryViewModel` for changes
     /// in state. The view updates the state of the `BasemapGalleryViewModel` in response to
     /// user action.
-    @ObservedObject private var viewModel: BasemapGalleryViewModel
+    @StateObject private var viewModel: BasemapGalleryViewModel
```

Comment:
Since the model is owned by the gallery view, I think this should be a `StateObject` instead.